Duplicati/Server/Program.cs

```diff
@@ -2,18 +2,22 @@
 using System.Collections.Generic;
 using System.Globalization;
 using System.Linq;
-using System.Text;
-using Duplicati.Library.Common;
-using Duplicati.Library.Common.IO;
-
+using Duplicati.Library.Common;
+using Duplicati.Library.Common.IO;
+
 namespace Duplicati.Server
 {
     public class Program
     {
+
+        private static readonly List<string> AlternativeHelpStrings = new List<string> { "help", "/help", "usage", "/usage", "--help" };
+
+        private static readonly List<string> ParameterFileOptionStrings = new List<string> { "parameters-file", "parameters-file", "parameterfile" };

```

Comment:
The first two entries here look identical.  Is this a typo?

Reply:
Actually, it looks like the previous implementation had this as well.  Perhaps we can use this opportunity to clean this up?

Reply:
I removed it, thanks for noticing.

Reply:
I _think_ it was supposed to be `parameter-file`, `parameters-file`, `parameterfile` to help users guessing their way with the commandline.

But if it did not work previously with the singular form, we can just remove it.